yuptoo/modifiers/transform_tags.py

```diff
@@ -7,6 +7,10 @@ def run(self, host: dict, transformed_obj: dict, **kwargs):
         if tags:
             tags_modified = False
             for tag in tags:
+
+                if len(str(tag['value'])) > 250:
+                    tag['value'] = "Original value exceeds 250 characters."
+
```

Comment:
Here we should add `tags_modified = True` because line 23 never execute if all the tag values are string.

Reply:
Good catch! :+1: